docs/content/latest/explore/ysql-language-features/tablespaces.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Tablespaces
+linkTitle: Tablespaces
+description: Tablespaces in YSQL
+headcontent: Tablespaces in YSQL
+image: /images/section_icons/secure/create-roles.png
+menu:
+  latest:
+    identifier: explore-ysql-language-features-tablespaces
+    parent: explore-ysql-language-features
+    weight: 300
+isTocNested: true
+showAsideToc: true
+---
+
+This document describes how use tablespaces in YSQL.
+
+## Overview
+
+YSQL tablespaces are entities that specify how data associated with them should be replicated and distributed across cloud, regions, and zones.
+
+In PostgreSQL, tablespaces are used for specifying a location on disk with options to control data access. That is, tablespaces group tables and indexes based on how you intend to store and access the data. For instance, you can choose to place heavily accessed smaller tables and indexes in SSD or “fast” storage compared to other tables.
+
+For YugabyteDB clusters, however, location does not pertain to disk locations. For a cloud-native distributed database, location pertains to the cloud, regions, and zone where the data is supposed to be. Therefore, although YSQL tablespaces are built on PostgreSQL tablespaces that allow you to specify placement of data at a table level, this placement information in YSQL defines the number of replicas for a table and index, as well as how they can be distributed across a set of cloud, regions, and zones. 
+
+## Defining a Tablespace
+
+You can define a tablespace using the following syntax:
+
+```sql
+CREATE TABLESPACE tablespace_name 
+  [ OWNER { username | CURRENT_USER | SESSION_USER } ]
+  WITH ( replica_placement = placement_policy_json );
+```
+
+*tablespace_name* represents the name of the tablespace to be created. 
+
+*username* represents the name of the user who will own the tablespace, with the name of the user executing the command being the default. Note that only superusers can create tablespaces and grant their ownership to other types of users. 
+
+*placement_policy_json* represents a JSON string that specifies the placement policy for this tablespace. The JSON structure contains two fields: `num_replicas` that defines the overall replication factor, and `placement_blocks` which is an array of tuples, with each tuple containing the keys `<”cloud”, “region”, “zone”, “min_num_replicas”>` whose values define a placement block. Typically the sum of `min_num_replicas` across all placement blocks is expected to be equal to `num_replicas`. The aggregate of `min_num_replicas` can be lesser than `num_replicas`, in which case the extra replicas are placed at the YB-Load balancer’s discretion. If you do not specify *replica_placement* for a tablespace, then associating the tablespace with a table or index affects how the data of the table is placed. 
```

Comment:
Actually we have to specify *replica_placement* for a tablespace. There is a corner case where if LOCATION is specified, you wouldn't need to, but I think in Explore section we don't need to add those confusing details that are not expected to be used at all.
Lets just remove this last line.

Reply:
Deleted the last sentence.